src/components/QuranReader/index.tsx

```diff
@@ -67,7 +69,8 @@ const QuranReader = ({
         isVerseData,
         isTafsirData,
         id,
-      ),
+        reciter: reciter.id,
+      }),
     verseFetcher,
     {
       initialData: isUsingDefaultTranslations && isUsingDefaultTafsirs ? initialData.verses : null, // initialData is set to null if the user changes/has changed the default translations/tafsirs so that we can prevent the UI from falling back to the default translations while fetching the verses with the translations/tafsirs the user had selected and we will show a loading indicator instead.
```

Comment:
You'll have to set initialData to null when we're not using the default Reciter. will explain why over a call iA